src/field.cpp

```diff
@@ -479,6 +428,114 @@ If you need to insert a new field behavior per unit time add a case statement in
 bool map::process_fields_in_submap( submap *const current_submap,
                                     const int submap_x, const int submap_y, const int submap_z )
 {
+    const auto get_neighs = [this]( const tripoint &pt ) {
```

Comment:
Please either change this to "neighbors", or use it to retrieve horse sound effects, I'm fine with either one ;)